cmd/connector/flags.go

```diff
@@ -40,4 +40,9 @@ var (
 		Usage: "Option for specifying db mode. Available options: `full-persister`, `import-db`, `optimized-persister`",
 		Value: "full-persister",
 	}
+
+	enableGrpcServer = cli.BoolFlag{
```

Comment:
do not forget to add `enableGrpcServer` into app cli.Flag list

Reply:
added